.gitignore

```diff
@@ -17,6 +17,8 @@ reports
 .history
```

Comment:
NRTTL:

Getting error, i tested like this:
1. `rm -rf node_modules yarn.lock`
2. `nvm install`
3. `npm i -g yarn`
4. `yarn install --registry=https://npm.lisk.com`

<img width="973" alt="image" src="https://github.com/LiskHQ/lisk-desktop/assets/8784876/e660e468-be95-43dc-afaf-af05cce0d516">

Reply:
you can just do yarn without all that.. it appears the error started around `liskhq` installation, which seems to be wrong due to the registry you mentioned, try with `just yarn`

Reply:
works now!